cli/src/main.rs

```diff
@@ -589,7 +589,7 @@ fn print_table(table: Vec<Vec<String>>) {
         let mut col_string = String::from("");
         for (i, len) in max_lengths.iter().enumerate() {
             if let Some(value) = row.get(i) {
-                col_string += &format!("{}{} ", value, " ".repeat(*len - value.len()),);
+                col_string = format!("{}{}{} ", col_string, value, " ".repeat(*len - value.len()),);
```

Comment:
This isn't really fixing the issue the lint complains about, which is the use of an additional format call

Reply:
Fixed, thanks